fastlane/spec/actions_specs/xcodebuild_spec.rb

```diff
@@ -27,6 +27,7 @@
             destination_timeout: 240,
             dry_run: true,
             enable_address_sanitizer: true,
+            enable_thread_sanitizer: true,
```

Comment:
this use cases seems a bit weird as if I understand `thread`  and `address sanitizer` can't be enabled together.